components/automate-ui/src/app/pages/+compliance/+credentials/components/credentials-list-row.html

```diff
@@ -6,17 +6,17 @@
   (deleteClicked)="deleteCredential.emit(credential)"
   objectAction="Remove">
 </app-delete-object-modal>
-<chef-tr>
-  <chef-td>
+<chef-table-row>
+  <chef-table-cell>
     <a [routerLink]="['/settings', 'node-credentials', credential.id, 'edit']">
       {{ credential.name }}
     </a>
-  </chef-td>
-  <chef-td>{{ credentialsLogic.formatKeyType(credential.type) }}</chef-td>
-  <chef-td>{{ getLastModified(credential.last_modified) }}</chef-td>
-  <chef-td class="three-dot-column">
+  </chef-table-cell>
+  <chef-table-cell>{{ credentialsLogic.formatKeyType(credential.type) }}</chef-table-cell>
+  <chef-table-cell>{{ getLastModified(credential.last_modified) }}</chef-table-cell>
+  <chef-table-cell class="three-dot-column">
     <mat-select panelClass="chef-control-menu">
       <mat-option (onSelectionChange)="startCredentialDelete()" data-cy="delete">Remove Credential</mat-option>
     </mat-select>
-  </chef-td>
-</chef-tr>
+  </chef-table-cell>
+</chef-table-row>
```

Comment:
this was scope creep - but i'm going to move forward with this type of swapping on its own branch.